Content.Client/PlantAnalyzer/UI/PlantAnalyzerWindow.xaml

```diff
@@ -0,0 +1,40 @@
+<DefaultWindow xmlns="https://spacestation14.io"
```

Comment:
should probably use FancyWindow to look better

Reply:
Not much difference, but it's using FanyWindow now.